forte/data/multi_pack.py

```diff
@@ -14,10 +14,14 @@
 
 import copy
 import logging
+
 from pathlib import Path
 from typing import Dict, List, Set, Union, Iterator, Optional, Type, Any, Tuple
 
+import jsonpickle
+
 from sortedcontainers import SortedList
+from packaging.version import Version
```

Comment:
`packaging` is not part of the standard library. We should add it to https://github.com/asyml/forte/blob/master/requirements.txt . It currently works in test cases because we have it should be a dependency of `pip` or `setuptools.

Reply:
Yes this line of packaging is now added to the requirements.txt